src/cpuinfo/SDL_cpuinfo.c

```diff
@@ -1025,6 +1025,13 @@ SDL_GetSystemRAM(void)
             }
         }
 #endif
+#ifdef __VITA__
+        if (SDL_SystemRAM <= 0) {
+            /* Vita has 512MiB on SoC, that's split into 256MiB(+109MiB in extended memory mode) for app
+               +26MiB of physically continuous memory, +112MiB of CDRAM(VRAM) + system reserved memory. */
+            SDL_SystemRAM = 536870912;
```

Comment:
Wouldn't it be better to use `sceKernelGetFreeMemorySize`? https://github.com/vitasdk/vita-headers/blob/master/include/psp2/kernel/sysmem.h#L135

Reply:
Nah, afaiu, that returns total installed mem, not available to app.